package.json

```diff
@@ -142,5 +142,37 @@
     "hooks": {
       "pre-commit": "lint-staged"
     }
+  },
+  "plugins": [
+    "@semantic-release/commit-analyzer",
+    "@semantic-release/release-notes-generator",
+    "@semantic-release/npm",
+    "@semantic-release/git"
+  ],
+  "release": {
+    "branches": [
+      "main"
```

Comment:
Did you rename the default branch to `main`?

Reply:
ahh missed it, on it

Reply:
[ready for review!](https://github.com/ratehub/ratehub-express-gateway/pull/16)